styles.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Montserrat", Arial, Helvetica;
+}
+
+li, a {
+  font-weight: 500;
+  font-size: 12px;
```

Comment:
If I do this, I will then have to explicitly state the font sizes for H1, H2, H3 and p.  Won't this require more code?

Reply:
Not really, you can just leave the h1 elements at it is but not style every li element as 12px in general. Why did you choose 12px for all list and anchor elements in the first place?

what I meant:
```
.text-sm{
  font-size: 8px;
}

.text-l{
  font-size: 16px;
}

.text-xl{
  font-size: 20px;
}
```

```
<li class="text-l">
...
</li>
```

Reply:
I chose 12px because from my initial estimation of your example I should  be following it looked like 12px.  I know I should have asked font size specifications before beginning.  I will make sure to do this next time.  Now that we are on the topic, can you please specify what font sizes you want for H1, H2, H3, p, li, buttons, inputs and a?